src/main/js/main_html5.js

```diff
@@ -562,6 +568,60 @@ require("../../../html5-common/js/utils/environment.js");
     }, this);
 
     /**
+     * Callback for when a closed caption track cue has changed.
+     * @private
+     * @method OoyalaVideoWrapper#onClosedCaptionCueChange
+     * @param {object} event The event from the cue change
+     */
+    var onClosedCaptionCueChange = _.bind(function(event) {
+      var cueText = "";
+      if (event && event.currentTarget && event.currentTarget.activeCues) {
+        for (var i = 0; i < event.currentTarget.activeCues.length; i++) {
+          if (event.currentTarget.activeCues[i].text) {
+            cueText += event.currentTarget.activeCues[i].text + " ";
+          }
+        }
+      }
+      raiseClosedCaptionCueChanged(cueText);
+    }, this);
+
+    /**
+     * Workaround for Firefox only.
+     * Check for active closed caption cues and relay them to the controller.
+     * @private
+     * @method OoyalaVideoWrapper#checkForClosedCaptionsCueChange
+     */
+    var checkForClosedCaptionsCueChange = _.bind(function() {
+      var cueText = "";
+      if (_video.textTracks) {
+        for (var i = 0; i < _video.textTracks.length; i++) {
+          if (_video.textTracks[i].activeCues) {
+            for (var i = 0; i < _video.textTracks[i].activeCues.length; i++) {
+              if (_video.textTracks[i].activeCues[i].text) {
+                cueText += _video.textTracks[i].activeCues[i].text + " ";
```

Comment:
I believe spaces because I saw multiple active cues in the live asset that actually belonged to the same sentence.